biggus/tests/unit/test__unary_fn_wrapper.py

```diff
@@ -0,0 +1,111 @@
+        wrapped_fn = biggus._unary_fn_wrapper('my_module.my_function',
+                                              lambda a: a)
+        self.assertEqual(wrapped_fn.__name__, (lambda a: a).__name__)
+
+    def test_given_fn_name(self):
+        wrapped_fn = biggus._unary_fn_wrapper('my_module.my_function',
+                                              lambda a: a, fn_name='identity')
+        self.assertEqual(wrapped_fn.__name__, 'identity')
+
+    def test_masked_array_creates_elementwise(self):
+        wrapped_fn = biggus._unary_fn_wrapper('my_module.my_function',
+                                              lambda a: a + 10,
+                                              lambda a: a - 10,
+                                              fn_name='identity')
+        self.assertIsInstance(wrapped_fn(np.array([1])), biggus._Elementwise)
+
+    def test_unexpected_n_arguments(self):
+        wrapped_fn = biggus._unary_fn_wrapper('my_module.my_function',
+                                              lambda a: a + 10,
+                                              fn_name='identity')
+        # TODO: It would be good if this were not called "wrapped_function".
+        if sys.version_info[0] == 2:
+            msg = 'wrapped_function\(\) takes exactly 1 argument \(2 given\)'
+        else:
+            msg = ('wrapped_function\(\) takes 1 positional argument but 2 '
+                   'were given')
+        with self.assertRaisesRegexp(TypeError, msg) as ex:
+            wrapped_fn(np.array([-5, 2]), np.array([-5, 2]))
+
+    def test_ndarray_expected_values(self):
+        wrapped_fn = biggus._unary_fn_wrapper('my_module.my_function',
+                                              lambda a: a + 10,
+                                              fn_name='identity')
+        assert_array_equal(wrapped_fn(np.array([-5, 2])).ndarray(),
+                           [5, 12])
+
+    def test_masked_array_expected_values(self):
+        wrapped_fn = biggus._unary_fn_wrapper('my_module.my_function',
+                                              lambda a: a + 10,
+                                              lambda a: a - 10,
+                                              fn_name='identity')
+        assert_array_equal(wrapped_fn(np.array([-5, 2])).masked_array(),
+                           [-15, -8])
+
+    def test_masked_array_undefined(self):
+        wrapped_fn = biggus._unary_fn_wrapper('my_module.my_function',
+                                              lambda a: a + 10,
+                                              fn_name='identity')
+        msg = 'No <lambda> operation defined for masked arrays.'
+        with self.assertRaisesRegexp(TypeError, msg):
+            assert_array_equal(wrapped_fn(np.array([-5, 2])).masked_array(),
+                               [-15, -8])
+
+
+class Test_wrapped_functions(unittest.TestCase):
+    def test_all_fns(self):
+        fns_to_test = ['isreal', 'iscomplex', 'isinf', 'isnan', 'signbit']
```

Comment:
And once more, see previous comments on the reasoning for the contents of this list.